src/App.jsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
+import Messages from "./pages/Messages";
 
 const App = () => {
   return (
     <>
-      <h1 className="text-4xl font-bold">React Boilerplate</h1>
+      {/* <h1 className="text-4xl font-bold">React Boilerplate</h1> */}
+      <Messages />
```

Comment:
It's best not to include the Messages component at the app level, especially if you're planning to have multiple components or pages in your application. Instead, consider setting up a routing configuration. If the Messages component is a distinct page, assign it its own route. If it's not a standalone page but part of another page, simply integrate it into the specific page that should have a defined route

Reply:
I have done this only because we split up the work and I just wanted to see the components I developed on screen. We will have routing configuration. You can see it in Samir's branch.